index.html

```diff
@@ -1362,12 +1362,12 @@ <h6 data-lang="zh-hant">發表對本文檔的建議</h6>
           <span data-lang="zh-hant">點號</span>
         </h5>
 
-        <p data-lang="en">Pause or stop punctuation marks are used to indicate  pauses or the end of a sentence. Some of the pause or stop punctuation marks appear within a sentence, such as the slight-pause comma, comma, semicolon, colon, etc., while others appear at the end of a sentence, such as period, question mark and exclamation mark.</p>
+        <p data-lang="en">Pause or stop punctuation marks are used to indicate pauses or the end of a sentence. Some of the pause or stop punctuation marks appear within a sentence, such as the slight-pause comma, comma, semicolon, colon, etc., while others appear at the end of a sentence, such as the period, question mark and exclamation mark.</p>
         <p data-lang="zh-hans">点号相对于标号，用于表示语句的停顿或暂停。分为句内点号，如顿号、逗号、分号、冒号等；以及句末点号，如句号、问号、感叹号等。</p>
         <p data-lang="zh-hant">點號相對於標號，用於表示語句的停頓或暫停。分為句內點號，如頓號、逗號、分號、冒號等；以及句末點號，如句號、問號、驚嘆號等。</p>
         <ol>
           <li>
-            <p data-lang="en"><span class="leadin">Ideographic full stops, fullwidth commas and ideographic comma.</span></p>
+            <p data-lang="en"><span class="leadin">Ideographic full stops, full-width commas and ideographic comma.</span></p>
```

Comment:
I'd prefer being consistent with Unicode terms. Both [UCD](http://ftp.unicode.org/Public/10.0.0/ucd/Blocks.txt) and [UAX #11](https://www.unicode.org/reports/tr11/) use the terms "fullwidth" and "halfwidth" without hyphens.

Reply:
I agree. See http://www.unicode.org/versions/Unicode10.0.0/ch18.pdf section 18.5.